test/Kozani/KozaniManagerRuntimeTests/KozaniManagerRuntimeTests.cpp

```diff
@@ -16,9 +16,9 @@ namespace KozaniManagerTestPackage
 
 namespace Test::KozaniManagerRuntimeTests
 {
-    struct MyKozaniStatusCallback : winrt::implements<MyKozaniStatusCallback, IKozaniStausCallback, winrt::Windows::Foundation::IInspectable>
+    struct MyKozaniStatusCallback : winrt::implements<MyKozaniStatusCallback, IKozaniStatusCallback, winrt::Windows::Foundation::IInspectable>
     {
-#pragma region IKozaniStausCallback_methods
+#pragma region IKozaniStatusCallback_methods
```

Comment:
Does the region name need to be 1 word?

(been a while since I used this pragma)

Reply:
it can have space.